src/main/java/com/google/sps/utils/GameUtils.java

```diff
@@ -20,6 +20,7 @@
 import com.google.appengine.api.datastore.EntityNotFoundException;
 import com.google.appengine.api.datastore.KeyFactory;
 import com.google.appengine.api.datastore.Key;
+import com.google.sps.utils.UserUtils;
```

Comment:
This adds UserUtils but GameUtils doesn't use UserUtils

Reply:
Yep, I took it out!